### ISSUE TITLE
Keywords do not respect capitalization

### ISSUE THREAD

**Issue by MarkWieczorek** [trutzig89182/Mastodon-Sociologists#93]
The keywords are all listed in lower case. Though this might make sense for most cases, it is sometimes useful to allow for upper case words. Some examples are: 
* Country names
* Organizations
* Acronyms
* Phases (as opposed to individual keywords)

I'm not sure if there is an easy way to fix this, but I just thought that I would bring it up...

**Comment by trutzig89182** [comment]
Hey @MarkWieczorek , sorry for the late reply, I saw your issue only now. I decided to use only lower-case because I wanted to make sure that keywords are interpreted as being the same, even if they differ in upper/lower case letters (so when you sort by keywords, there is no difference between "political sociology" and "Political Sociology". 
Of course it would be possible to still print them out in the way they have been commited. This might only look a bit more messy.

I see your point generally, but I am not sure if it has been a concrete problem so far. Has it been a problem for you? 

As the idea was to provide a way to enter research interests or subdisciplines, I am not sure if entering Organisations or Acronyms would make very much sense. But I am open to be convinced of the contrary. 

Again, sorry for the late reply.
